### PR TITLE
Add spacing in CallToAction banner

### LINKED ISSUE #3690: Small spacing issue with the CallToAction banner on the control plane dashboard
When I install the linkerd control plane in my cluster, and open the dashboard before injecting any of my pods, the "Control Plane" page shows me a call to action that looks like this:

image

The banner is sitting flush with the headers below it, but I suspect it would be better to space it out a bit.

### PR DESCRIPTION
**Pull request by aliariff**
The call to action banner in the control plane page is missing some spacing.

The CSS is defined but not yet used. So the solution is to add the class name to the corresponding banner.

After its merged, the banner will have more space.

Before:
image

After:
image

Fixes #3690

Signed-off-by: Ali Ariff <ali.ariff12@gmail.com>

### CODE REVIEW

**Comment by Pothulapati** [comment]
This looks good, but looks like the tests were a bit flaky. Try amending the commit, so that the CI Checks are re-run?

**Comment by aliariff** [comment]
@Pothulapati CI passed now.

File: web/app/js/components/CallToAction.jsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ const CallToAction = ({ resource, numResources }) => {
       <Typography>The service mesh was successfully installed!</Typography>
       <Stepper
         activeStep={lastStep}
+        className={classes.instructions}
```
Comment (scottcarol) [inline web/app/js/components/CallToAction.jsx]: This is a great fix, thanks! It looks like there are still two un-used `styles` specifications, `root` and `button`. Would you mind removing those from the component as well before we merge this in?

Comment (aliariff) [reply web/app/js/components/CallToAction.jsx]: Yes sure

**Pull request merged**